Pcap++/src/DpdkDevice.cpp

```diff
@@ -792,11 +792,6 @@ uint16_t DpdkDevice::receivePackets(MBufRawPacketVector& rawPacketsArr, uint16_t
 	//the following line trashes the log with many messages. Uncomment only if necessary
 	//PCPP_LOG_DEBUG("Captured %d packets", numOfPktsReceived);
 
-	if (unlikely(numOfPktsReceived <= 0))
-	{
-		return 0;
-	}

```

Comment:
Maybe this is not significant, but if `numOfPktsReceived == 0` we can return here and don't need to run `clock_gettime` and a for loop with zero iterations.

The same comment applies to the rest of the similar changes in this PR

Reply:
clock_gettime calls are very cheap, but I agreed, every little bit counts.